modules/FeaturedCategories/FeaturedCategories.tsx

```diff
@@ -0,0 +1,28 @@
+import type { Locale } from '@prezly/theme-kit-nextjs';
+
+import { app } from '@/adapters/server';
+
+import * as ui from './ui';
+
+interface Props {
+    localeCode: Locale.Code;
+}
+
+export async function FeaturedCategories({ localeCode }: Props) {
+    const categories = await app().categories();
+    const translatedCategories = await app().translatedCategories(
+        localeCode,
+        categories.filter((i) => i.is_featured),
```

Comment:
Probably should also filter out categories that have no stories like we do in [other places](https://github.com/prezly/theme-nextjs-bea/blob/main/modules/Header/Header.tsx#L22).

Reply:
2d932f46